libcasr/src/stacktrace.rs

```diff
@@ -31,18 +38,7 @@ lazy_static::lazy_static! {
 macro_rules! init_ignored_frames {
     ( $( $x:expr ),* ) => {
         {
-            let (funcs, files): (Vec<_>, Vec<_>) = [$($x,)*].iter().map(|&x|
-                match x {
-                    "python" => (STACK_FRAME_FUNCTION_IGNORE_REGEXES_PYTHON, STACK_FRAME_FILEPATH_IGNORE_REGEXES_PYTHON),
-                    "rust" => (STACK_FRAME_FUNCTION_IGNORE_REGEXES_RUST, STACK_FRAME_FILEPATH_IGNORE_REGEXES_RUST),
-                    "cpp" => (STACK_FRAME_FUNCTION_IGNORE_REGEXES_CPP, STACK_FRAME_FILEPATH_IGNORE_REGEXES_CPP),
-                    "go" => (STACK_FRAME_FUNCTION_IGNORE_REGEXES_GO, STACK_FRAME_FILEPATH_IGNORE_REGEXES_GO),
-                    "java" => (STACK_FRAME_FUNCTION_IGNORE_REGEXES_JAVA, STACK_FRAME_FILEPATH_IGNORE_REGEXES_JAVA),
-                    &_ => (["^[^.]$"].as_slice(), ["^[^.]$"].as_slice()),
-                }
-            ).unzip();
-           *STACK_FRAME_FUNCTION_IGNORE_REGEXES.write().unwrap() = funcs.concat().iter().map(|x| x.to_string()).collect::<Vec<String>>();
-           *STACK_FRAME_FILEPATH_IGNORE_REGEXES.write().unwrap() = files.concat().iter().map(|x| x.to_string()).collect::<Vec<String>>();
+            <Stacktrace as Filter>::init_frame_filter(&[$($x,)*]);
```

Comment:
In function `filter()` from `impl Filter for Stacktrace` we need to check that `STACK_FRAME_FUNCTION_IGNORE_REGEXES` and `STACK_FRAME_FUNCTION_IGNORE_REGEXES` are not empty. Otherwise we get panic at `attempt to subtract with overflow` if we don't use `init_ignored_frames!` macro with some langs.